app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  after_initialize :update_likes_counter
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes_counter.to_i + 1)
+  end
```

Comment:
- [ OPTIONAL] I recommend keeping the update likes method private since you used one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new like has been saved to database, so I think it would be better if you keep as you make It first 💪🏻✅👏🏻💯🔥